compiler/testData/integration/smoke/helloAppSuspendParameterlessMainIR/hello.run.expected

```diff
@@ -0,0 +1,4 @@
+OUT:
+OK
```

Comment:
This line is not printed in both CI when I run locally. The same happens in `helloAppSuspendMainInMultifileIr`. `helloAppSuspendMainIR`, however, runs OK when I run as a test, but does not print "${args[0]$args[1]}" when I compile the file and run it from CLI.

Reply:
Yeah, this is but a WIP to include the tests in the suite - I haven't worked out what's wrong yet. I think it's the interaction of the testing setup for suspend and the new coroutine backend, but not sure.

Ideas?